main_test.go

```diff
@@ -15,6 +15,16 @@ func TestBaseCommandsAreRegistered(t *testing.T) {
 	baseManager := cmd.BuildBaseManager("tranor", "", "", nil)
 	manager := buildManager("tranor")
 	for name, expectedCommand := range baseManager.Commands {
+		var skip bool
+		for _, c := range baseCommandsToRemove {
```

Comment:
Maybe we should define our own commandsToRemove for the tests, then we can use them both here and below. If we did that, then it would be easy to see that the test's commandsToRemove was incomplete.